src/signals/incident/components/IncidentForm/index.test.js

```diff
@@ -410,4 +410,68 @@ describe('<IncidentForm />', () => {
       })
     })
   })
+
+  describe('handleSubmit', () => {
+    it('should trigger description and source validation', async () => {
+      const fieldConfig = {
+        controls: {
+          $field_0: {
+            isStatic: false,
+            render: IncidentNavigation,
+          },
+          description: {
+            meta: {
+              label: 'description',
+            },
+            options: {
+              validators: ['required'],
+            },
+            render: FormComponents.TextInput,
+          },
+          source: {
```

Comment:
When logged in, source is not a hidden field, otherwise hidden. I choose to only simulate the logged in version, only then you could have an error.